packages/pieces/google-drive/src/lib/action/search-folder.action.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { googleDriveAuth } from '../../index';
+import { Property, createAction } from "@activepieces/pieces-framework";
+import { google } from 'googleapis';
+import { OAuth2Client } from 'googleapis-common';
+
+export const googleDriveSearchFolder = createAction({
+    auth: googleDriveAuth,
+    name: 'search-folder',
+    displayName: 'Search folder',
+    description: 'Search folder from a Google Drive folder',
+    props: {
+        query: Property.ShortText({
+            displayName: 'Name',
+            description: 'Name to search for',
+            required: true,
+        }),
+    },
+    async run (context) {
+
+        console.log(context.auth);
```

Comment:
Can you remove this as it is sensitive information and it shouldn't show in logs

Reply:
Oops,I will remove it.